tools/dashboard.go

```diff
@@ -16,20 +16,56 @@ type GetDashboardByUIDParams struct {
 
 func getDashboardByUID(ctx context.Context, args GetDashboardByUIDParams) (*models.DashboardFullWithMeta, error) {
 	c := mcpgrafana.GrafanaClientFromContext(ctx)
-	
+
 	dashboard, err := c.Dashboards.GetDashboardByUID(args.UID)
 	if err != nil {
 		return nil, fmt.Errorf("get dashboard by uid %s: %w", args.UID, err)
 	}
 	return dashboard.Payload, nil
 }
 
+type UpdateDashboardParams struct {
+	Dashboard map[string]interface{} `json:"dashboard" jsonschema:"required,description=The full dashboard JSON"`
+	FolderUID string                 `json:"folderUid" jsonschema:"optional,description=The UID of the dashboard's folder"`
+	IsFolder  bool                   `json:"isFolder" jsonschema:"optional,description=Whether the dashboard is a folder"`
```

Comment:
I don't think we should expose this option to the model - it feels like a quirk of Grafana's dashboard API more than anything else. If we want the model to be able to create folders we should have a separate tool which just calls this API endpoint with `isFolder: true`.